src/app/modules/project/overview/overview.component.ts

```diff
@@ -239,6 +239,11 @@ export class OverviewComponent implements OnInit {
     }
 
     this.currentSearchInput = value;
+
+    // Reset the current page so we are sure we retrieve all the projects in the database and not
+    // Just the ones on the current page. If the query is empty we should search for the projects
+    // On the first page only
+    this.currentPage = value.trim() === '' ? 1 : null;
```

Comment:
We have discussed with @Brend-Smits that it is better for the UX if we continue on the same page the search was initially performed on instead of always going back to the first page when the search query is cleared. So, could you implement this?